project1/templates/Registration.html

```diff
@@ -24,7 +24,7 @@ <h1 style="text-light">Project 1</h1>
         </br>
             <h3 class="text-primary">Registration Page</h3>
         </br>
-            <form action="http://localhost:5000/register" method = "POST">
+            <form>
```

Comment:
You should only remove http://localhost:5000. rest all should be there. <form action = "/register" method = "POST">